tests/base.py

```diff
@@ -485,6 +485,8 @@ def setUp(self, config_dict='config_dict'):
                          'subType': 'json'},
                         {'id': '133', 'key': 'true_object', 'defaultValue': '{"true_test": 23.54}', 'type': 'json'},
                     ],
+                    'delivery_rules': [],
+                    'experiment_rules': [],
```

Comment:
Do we need add them here? These are for OptimizelyConfig only, not for projectConfig. 
Same for other "devliery_rules" and "experiment_rules" below.

Reply:
It looks like a wrong place to add them. We can discuss.